### PR TITLE
[CPUCache]Enable cpu cache for LRN

### PR DESCRIPTION
**Pull request by mangguo321**
### Details:
 - *Enable cpu cache for LRN*
 - *...*

### Tickets:
 - *unknown*

### CODE REVIEW

**Comment by dmitry-gorokhov** [comment]
@mandrono Please review the PR

File: src/plugins/intel_cpu/src/nodes/mkldnn_lrn_node.cpp [hunk 1]
```diff
@@ -8,10 +8,54 @@
...
+    return seed;
+}
+
+bool LrnKey::operator==(const LrnKey &rhs) const {
+    bool retVal = true;
+    if (inp0 != rhs.inp0) {
+        retVal = retVal && inp0 && rhs.inp0 && inp0->getDnnlDesc() == rhs.inp0->getDnnlDesc();
+    }
+
+    retVal = retVal && implType == rhs.implType && alg == rhs.alg && alg == rhs.alg && size == rhs.size && k == rhs.k &&
```
Comment (mandrono) [inline src/plugins/intel_cpu/src/nodes/mkldnn_lrn_node.cpp:53]:
```suggestion
    retVal = retVal && implType == rhs.implType && alg == rhs.alg && size == rhs.size && k == rhs.k &&
```

**Review by mandrono — APPROVED** [review]

File: src/plugins/intel_cpu/src/nodes/mkldnn_lrn_node.cpp (hunk 1 above)
Comment (dmitry-gorokhov) [reply src/plugins/intel_cpu/src/nodes/mkldnn_lrn_node.cpp:53]: @mangguo321 please add the fix in https://github.com/openvinotoolkit/openvino/pull/9549

Comment (mangguo321) [reply src/plugins/intel_cpu/src/nodes/mkldnn_lrn_node.cpp:53]: OK. Thanks @dmitry-gorokhov

**Pull request merged**